CefSharp.props

```diff
@@ -31,6 +33,19 @@ if $(PlatformName) == x86 (
   call "$(DevEnvDir)..\..\VC\Auxiliary\Build\vcvars64.bat"
   editbin /TSAWARE "$(TargetPath)"
 )
+]]>
+    </CefSharpBrowserSubprocessPostBuildEvent>
+	
+    <CefSharpBrowserSubprocessPostBuildEvent Condition="'$(VisualStudioVersion)'=='16.0'">
```

Comment:
It was only the `PlatformToolset` that needed it's own entry. Fine to leave for now as it will be cleaned up when I remove `VS2013` support (when full `VS2019` support is added)